README.rst

```diff
@@ -35,8 +35,9 @@
 .. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
         :target: https://github.com/regro/regolith/issues
 
-A Group Content Management System
+A research group content management system
 
+* LONGER DESCRIPTION HERE
```

Comment:
going to be populated in the next PR?

Reply:
Is there an existing README, possibly with a different name? I honestly don't remember. If there is, that will likely have the longer description

Reply:
older README.rst did not have a longer description... I'll create an issue and we can populate this later?